docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1088,6 +1088,89 @@ Finally, if a container image was created during the build (by using including t
 
 As is the case with `@NativeImageTest`, this is a black box test that supports the same set features and has the same limitations.
 
+=== Code Coverage for Integration Tests
```

Comment:
Actually can you move this to https://github.com/quarkusio/quarkus/blob/c28a2276d9fbdc5c438c19b39b5b357daee58424/docs/src/main/asciidoc/tests-with-coverage.adoc

Otherwise looks good.

Reply:
Change pushed.